db/migrate/20230531175434_create_reviews.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReviews < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reviews do |t|
+      t.float :rating
```

Comment:
I would think the rating should be an integer.  4/5 would look better than potentially 4.37 rating